pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/MailboxReceiveOperatorTest.java

```diff
@@ -64,40 +69,35 @@ public class MailboxReceiveOperatorTest {
   private ReceivingMailbox _mailbox1;
   @Mock
   private ReceivingMailbox _mailbox2;
-  private StageMetadata _stageMetadataBoth;
-  private StageMetadata _stageMetadata1;
 
-  @BeforeMethod
+  @BeforeClass
```

Comment:
was this change related to Ser/De or just test optimization?

Reply:
Moving the per method setup to per class is test optimization